examples/requests-missing-timeout.py

```diff
@@ -1,27 +1,68 @@
+import httpx
 import requests
 import not_requests
 
+# Errors
 requests.get('https://gmail.com')
 requests.get('https://gmail.com', timeout=None)
-requests.get('https://gmail.com', timeout=5)
```

Comment:
Please don't remove these. They verify there's no regression when someone passes a timeout value

Reply:
They have not been removed but moved under "Okay" section below, since those cases did not trigger violations.